Source/PsWebServer/Private/PsWebServerHandlerImpl.cpp

```diff
@@ -166,7 +197,7 @@ FString FPsWebServerHandlerImpl::GetResponseData(const FGuid& RequestId, bool bT
 		Context.CancellationSource.Cancel();
 
 		const auto Handler = OwnerHandler.Load();
-		if (Handler && !Handler->IsPendingKill())
+		if (Handler && Handler == nullptr)
```

Comment:
It would be better to use `IsValid(Object)` as a replacement for `Handler && !Handler->IsPendingKill()`.